### PR TITLE
Enable path point forces and visualization on phyiscal frames

### PR DESCRIPTION
**Pull request by aseth1**
Path point associated with `WrapObjects` where applying forces directly to the point as if it was expressed in the `MobilizedBody`. In the case of `PhysicalOffsetFrame` we have to account for the potential series of offset transforms by using `findTransformInBaseFrame()` and using it to transform the point in to the base mobilized body frame. This correction was necessary for both applying forces to the points and visualizing them correctly.

### CODE REVIEW

File: OpenSim/Simulation/Model/Force.cpp [hunk 1]
```diff
@@ -137,25 +137,32 @@ double Force::computePotentialEnergy(const SimTK::State& state) const
 //-----------------------------------------------------------------------------
 // METHODS TO APPLY FORCES AND TORQUES
 //-----------------------------------------------------------------------------
-void Force::applyForceToPoint(const SimTK::State &s, const PhysicalFrame &aBody, const Vec3& aPoint, 
-                                    const Vec3& aForce, Vector_<SpatialVec> &bodyForces) const
+void Force::applyForceToPoint(const SimTK::State &s, const PhysicalFrame &frame,
+                              const Vec3& point, const Vec3& forceInG,
```
Comment (tkuchida) [inline OpenSim/Simulation/Model/Force.cpp]: `point` -> `pointInF` would be clearer, and would self-document L145.

**Comment by chrisdembia** [comment]
I don't really understand this stuff but the tests looks good. Might be good to test with the hopper device (create a new branch in which both the hopper branch and this branch are combined) to see if it fixes the issue for the hopper example.

Also, maybe it's fine to merge the hopper example into master now, but then we can still keep working on it on the tkuchida-updateHopperExample branch. But this way, anything that gets merged into master could be tested against the hopper example.

**Comment by tkuchida** [comment]
`</review>` LGTM aside from the one comment. I'll test on the hopper branch now…

**Comment by tkuchida** [comment]
:+1: Great! Successfully wrapped `PathActuator`s around a `WrapCylinder` that lives on a `PhysicalOffsetFrame`. I suggest we merge this PR. I'll then merge master into the hopper branch and apply my local changes, whereupon we can merge the hopper PR #1031.

**Comment by chrisdembia** [comment]
Sweet. @aseth1 says "thank you!" If we wait 10 minutes, we can make sure at least one of the `travis-ci/pr` builds passes.

**Pull request merged**